feat: hydra JWK caching (#2299)
ory/docs: docs/hydra/jwks.mdx

```

For more information on how to create a custom keyset, see the [ory create jwk](../cli/ory-create-jwk.md) command.

## Caching of JWKs

> Ory Hydra OEL supports in-memory caching of JWKs to improve performance and reduce latency. It is configurable via the following

Reviewer: ```suggestion
The Ory Hydra Enterprise License (OEL) supports in-memory caching of JWKs to improve performance and reduce latency. It is configurable via the following
```

options:

- `db.jwk_cache.enabled` governs whether the cache is enabled or not
- `db.jwk_cache.size` is the maximum number of keys to be cached
- `db.jwk_cache.ttl` is the time-to-live for cached keys

> It is recommended to set a cache TTL to force regular refreshes of the keys. Updates to keys are not actively propagated to the
> cache, so it is possible for tokens to be signed with rotated keys for the cache TTL duration. Clients are expected to accept old
> keys for a certain period of time.

Reviewer: ```suggestion
It's recommended to set a cache TTL to force regular refreshes of the keys. Updates to keys aren't propagated to the
cache, so it's possible for tokens to be signed with rotated keys for the cache TTL duration. Clients are expected to accept old
keys for a certain period of time.
```
